locales/hr.json

```diff
@@ -0,0 +1,17 @@
+{
+	  "unlocked_door": "Otključana vrata",
+    "locked_door": "Zaključana vrata",
+    "lock_door": "[E] Zaključaj vrata",
+    "unlock_door": "[E] Otključaj vrata",
+    "door_lock": "Lokot",
+    "passcode": "Lozinka",
+    "lockpick_broke": "Polomila Vam se brava",
+    "pick_lock": "Odaberi bravu",
+    "add_lock": "Dodaj lokot",
+    "edit_lock": "Uredi lokot",
+    "remove_lock": "Obriši lokot",
```

Comment:
Perhaps these should be "brava/bravu" instead of "lokot" as it's maybe a more known word? 🤷

Reply:
however you want